modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/Parameter.java

```diff
@@ -16,93 +16,113 @@
 
 package io.swagger.oas.annotations;
 
-import io.swagger.oas.annotations.enums.Explode;
-import io.swagger.oas.annotations.media.ArraySchema;
-import io.swagger.oas.annotations.media.Content;
-import io.swagger.oas.annotations.media.Schema;
-import io.swagger.oas.annotations.parameters.Parameters;
-
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Inherited;
-import java.lang.annotation.Repeatable;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+import io.swagger.oas.annotations.enums.Explode;
+import io.swagger.oas.annotations.media.ArraySchema;
+import io.swagger.oas.annotations.media.Content;
+import io.swagger.oas.annotations.media.Schema;
+
 /**
- * 
- *
- * 
+ * Describes a single operation parameter
  **/
-
-
 @Target({ ElementType.PARAMETER,
           ElementType.METHOD })
-@Repeatable(Parameters.class)
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface Parameter {
   /**
-   * the name of the parameter
+   * The name of the parameter.
+   * @return the parameter's name
    **/
   String name() default "";
 
   /**
-   * the location of the parameter.  Ignored when empty string
+   * The location of the parameter.  Possible values are "query", "header", "path" or "cookie".  Ignored when empty string.
+   * @return the parameter's location
    **/
   String in() default "";
 
   /**
    * Additional description data to provide on the purpose of the parameter
+   * @return the parameter's description
    **/
   String description() default "";
 
   /**
-   * Specifies that the parameter is not optional and must be present
+   * Specifies that the parameter is not optional and must be present.
+   * @return whether or not the parameter is required
    **/
   boolean required() default false;
 
   /**
-   * 
+   * Specifies that a parameter is deprecated and should be transitioned out of usage.
+   * @return whether or not the parameter is deprecated
    **/
   boolean deprecated() default false;
 
   /**
    * When true, allows sending an empty value.  If false, the parameter will be considered \&quot;null\&quot; if no value is present.  This may create validation errors when the parameter is required.
+   * @return whether or not the parameter allows empty values
    **/
   boolean allowEmptyValue() default false;
 
   /**
-   * 
+   * Describes how the parameter value will be serialized depending on the type of the parameter value. Default values (based on value of in): for query - form; for path - simple; for header - simple; for cookie - form.
+   * @return the style of the parameter
    **/
   String style() default "";
```

Comment:
Cool, I opened https://github.com/swagger-api/swagger-core/issues/2372 for the style enum investigation.   

Fixing the javadoc to clarify the exclusion with `content`